server/src/main/java/org/elasticsearch/action/admin/indices/rollover/TransportRolloverAction.java

```diff
@@ -124,7 +124,7 @@ protected void masterOperation(Task task, final RolloverRequest rolloverRequest,
         final String rolloverIndexName = indexNameExpressionResolver.resolveDateMathExpression(unresolvedName);
         MetaDataCreateIndexService.validateIndexName(rolloverIndexName, state); // will fail if the index already exists
         checkNoDuplicatedAliasInIndexTemplate(metaData, rolloverIndexName, rolloverRequest.getAlias());
-        IndicesStatsRequest statsRequest = new IndicesStatsRequest().indices(rolloverRequest.getAlias()).clear().docs(true);
+        IndicesStatsRequest statsRequest = new IndicesStatsRequest().indices(sourceIndexName).clear().docs(true);
```

Comment:
I wonder if there is a combination of indices options (`.indicesOptions(...)`) we can set that will allow us to still use the alias, I think setting `ignore_unavailable` to true may skip closed indices?

Reply:
This was a really good thought, `ignore_unavailable` looks like it might do the trick. I'll push a version that uses that shortly for a full CI run.